torch/_export/serde/schema.py

```diff
@@ -236,13 +236,19 @@ class InputToCustomObjSpec:
     custom_obj_name: str
 
 
+@dataclass
+class InputTokenSpec:
+    arg: TensorArgument
```

Comment:
why tokens are tensors instead of their own types?

Reply:
Mm... the tokens themselves are actually just tensors

Reply:
if they are tensors, do they have associated tensormetas, entries in the tensorvalues table, etc?

Reply:
They have associated tensor meta, but no entries in the tensorValues table since it doesn't matter what value they have -- they can just be dummy `tensor([])`

Reply:
I find this sort of weird…is there a specific reason why we want to make tokens into tensors instead of their own type?

Reply:
I think it was just easier to introduce them as tensors since tensors are already integrated into everything. I guess we can make it a special type -- is it easy to introduce a new type of input?

Reply:
It looks like:
1. Need to add a new IValue type (ivalue.h)
2. Need to add schema support, (schema_type_parser.cpp)
3. Need to add it to the export schema
5. Backends now need to implement it.

But I think it's a good idea. Re-using a tensor for this is hacky; it makes the operator schema misleading and creates a new special case that code that operates generically over tensors in the graph needs to be aware of.

@zou3519 thoughts?

Reply:
I updated the existing code to use a new TokenArgument rather than TensorArgument.
And as for adding the new IValue, we can discuss with Richard once hes back from his pto